### DISCUSSION TITLE
Gaming

### DISCUSSION THREAD

**Discussion by hernessteve** [NAIST-SE/AttractingContributors#385] [General]
Anyone playing video games?
What exactly?

**Comment by ellielo** [comment]
I am a fellow student and don't have too much playing video games. If i do have it (on a weekend mostly) than i prefer horrors. Troll Face Quest: Horror 2 from gamerdam [https://gamerdam.com/](https://gamerdam.com/) is one of the best horrors from my experience. If anyone has other recommendations i would gladly check them out too. Good music and cool graphics are essential components for a good game in my opinion.
